drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -3289,6 +3289,9 @@ RasterizerSceneGLES3::RasterizerSceneGLES3() {
 	// Quality settings.
 	use_physical_light_units = GLOBAL_GET("rendering/lights_and_shadows/use_physical_light_units");
 
+	positional_soft_shadow_filter_set_quality(RS::ShadowQuality(int(GLOBAL_GET("rendering/lights_and_shadows/positional_shadow/soft_shadow_filter_quality"))));
```

Comment:
```suggestion
	positional_soft_shadow_filter_set_quality((RS::ShadowQuality)(int)GLOBAL_GET("rendering/lights_and_shadows/positional_shadow/soft_shadow_filter_quality"));
```
Same below, matches other syntax and is more readable

Reply:
Agreed this reads better as a cast. FYI I copied these lines from servers\rendering\renderer_rd\renderer_scene_render_rd.cpp RendererSceneRenderRD::init